### PR TITLE
custom connector for certification  - nps.today

### PR DESCRIPTION
**Pull request by HazemSaeid**
- [X ] I attest that the connector works and I verified by deploying and testing all the operations.
- [ X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ X] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: certified-connectors/nps.today/readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (ptalrejagithub) [inline certified-connectors/nps.today/readme.md:1]: @HazemSaeid , can you please refer to this [Readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md) example. Readme.md is more for the open-source contributors and developers for building your connector.

File: certified-connectors/nps.today/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,788 @@
...
+                  "type": "string",
+                  "description": "PhoneNumber"
+                },
+                "ExternalId": {
+                  "type": "string",
+                  "description": "ExternalId"
+                },
+                "EmployeeExternalId": {
+                  "type": "string",
+                  "description": "EmployeeExternalId"
```
Comment (sriyen-msft) [inline certified-connectors/nps.today/apiDefinition.swagger.json:204]: @HazemSaeid The description values of properties should not be the property ids. Can you please provide valid description values for all the propeties.

Documentation - [Description values of properties](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/nps.today/apiDefinition.swagger.json (hunk 2 above)
Comment (HazemSaeid) [reply certified-connectors/nps.today/apiDefinition.swagger.json]: Resolved in latest commit

**Comment by HazemSaeid** [comment]
Hi, anything new?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@HazemSaeid There are many description values of properties need to be fixed, Please provide valid description values, they should be free from grammatical mistakes.

**Comment by HazemSaeid** [comment]
> @HazemSaeid There are many description values of properties need to be fixed, Please provide valid description values, they should be free from grammatical mistakes.

Thanks for the heads up. They are now fixed :)

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**